grafana-pd-dashboard.md

```
@@ -39,14 +39,14 @@ PD ダッシュボード メトリック項目の説明は次のとおりです
 
 ## オペレーター {#operator}
 
--   オペレータ作成のスケジュール: タイプごとに新しく作成されるオペレータの数
--   スケジュール演算子チェック: 種類ごとにチェックされる演算子の数。主に現在のステップが完了したかどうかをチェックし、完了している場合は次に実行するステップを返します。
--   スケジュールオペレータ終了: タイプごとに終了したオペレータの数
--   スケジュールオペレータのタイムアウト: タイプごとのタイムアウトオペレータの数
--   スケジュールオペレータの交代またはキャンセル: タイプごとに交代またはキャンセルされたオペレータの数
--   状態別のオペレーター数: 状態ごとのオペレーター数
--   オペレータ終了時間: 終了したオペレータの最大時間
--   オペレータステップの所要時間: 完了したオペレータステップの最大所要時間
+-   **Schedule operator create**: タイプごとに新しく作成されるオペレーターの数
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)